src/emulator/auth/state.ts

```diff
@@ -21,6 +21,8 @@ export class ProjectState {
   private userIdForProviderRawId: Map<string, Map<string, string>> = new Map();
   private refreshTokens: Map<string, RefreshTokenRecord> = new Map();
   private refreshTokensForLocalId: Map<string, Set<string>> = new Map();
+  private mfaEnrollmentIdsForLocalId: Map<string, Set<string>> = new Map();
```

Comment:
How do you feel about just dropping `mfaEnrollmentIdsForLocalId` altogether for this PR? I don't think it's read anywhere, and the same information can be easily retrieved from `user.mfaInfo` when we need it. This will make the code much much easier.

Reply:
I agree with the recommendation -- the thought had previously crossed my mind, and frankly, this is around primarily because I had based my initial implementation off of related code for managing provider id's.